content/docs/v3.4/benchmarks/etcd-2-2-0-benchmarks.md

```diff
@@ -71,3 +71,5 @@ The performance is calculated through results of 100 benchmark rounds.
 - Write QPS to cluster leaders seems to be increased by a small margin. This is because the main loop and entry apply loops were decoupled in the etcd raft logic, eliminating several blocks between them.
 
 - Write QPS to all members seems to be increased by a significant margin, because followers now receive the latest commit index sooner, and commit proposals more quickly.
+
+[hack-benchmark]: https://github.com/etcd-io/etcd/tree/master/hack/benchmark
```

Comment:
`master` <-- `v3.4.x` for the latest x patch release for pages under `/docs/v3.4`

Reply:
should this be v3.4, this looks like a 2.2 benchmark...

Reply:
I'd like to address this in https://github.com/etcd-io/website/issues/147